docs/_docs/tools/control-script.adoc

```diff
@@ -646,4 +646,208 @@ control.bat --change-tag _<new-tag>_
 ----
 --
 
+== Metric Command
+
+The metrics command prints out the value of a metric or metric registry provided in the parameters list. Use the `--node-id` parameter, If you need to get a metric from a specific node. Ignite selects a random node, if the `--node-id` is not set.
```

Comment:
"If you need to" -> "if you need to". The first letter needs to be in the lower case.

Reply:
Fixed.